libs/slam/async_slam/async_slam_localize.cpp

```
@@ -137,15 +137,20 @@ void AsyncSlam::LocalizeInMapCmd::Execute(AsyncSlam& async_slam, FrameId, const
   CALLBACK_AND_RETURN_IF(!result.slam_from->SelectHeadKeyframe(result.from_keyframe_id, timestamp_ns_), finish_cb_,
                          Pose, "Internal error during set head keyframe.");
 
-  const PoseGraphOptimizerOptions pgo_opt = async_slam.slam_->GetPoseGraphOptimizerOptions();
+  PoseGraphOptimizerOptions pgo_opt;
+  std::optional<std::vector<CameraId>> maybe_active_cameras;
```

Clang (14.0.6)

[warning] variable 'maybe_active_cameras' is not initialized

(cppcoreguidelines-init-variables)